Code/GraphMol/MolDraw2D/MolDraw2DSVG.cpp

```diff
@@ -276,9 +271,7 @@ void MolDraw2DSVG::drawEllipse(const Point2D &cds1, const Point2D &cds2) {
        << " rx='" << w / 2 << "'"
        << " ry='" << h / 2 << "'";
 
-  if (d_activeClass != "") {
-    d_os << " class='" << d_activeClass << "'";
-  }
+  outputClasses();
```

Comment:
I just checked the output from all of the tests and realized that this bit of code results in the following types of stuff showing up in the output SVG:
```
ry='3.78972'class='bond-13'
```
This causes at least some SVG renderers to fail.

The fix is to add a space at the end of the string being output on line 272:
```
       << " ry='" << h / 2 << "' ";
```

Reply:
Once this is fixed the PR is ready to merge.

Reply:
That is clearly an error. Now fixed.